src/index.html

```diff
@@ -9,10 +12,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/style.scss"
```

Comment:
The `href` attribute is pointing to an SCSS file. Please compile your SCSS into a CSS file and update the `href` to point to the compiled CSS file.